### DISCUSSION TITLE
Bug Report and Seeking Help

### DISCUSSION THREAD

**Discussion by ioudove** [openai/whisper#2141] [Q&A]
1、I am running Whisper version 20231117 in Colab, using the Large model. I submitted a video file with a duration of 1 hour and 36 minutes. When playing the video, I noticed that the subtitles generated by Whisper are not synchronized with the spoken content in the video. The subtitles currently displayed should have appeared earlier in the video to be correct. I hope this bug can be fixed in the next version.

2、I am currently struggling with this [issue](https://github.com/openai/whisper/discussions/1962), but I have no programming knowledge. Even though Purfview has provided the correct solution and steps, I still haven't succeeded when attempting their method. I need to resolve this problem. Can someone please help me? If you are willing to assist, please simply tell me what I need to do. I sincerely appreciate any help you can provide.（Additionally, I have found that for the same video, if I use the `medium` model, this issue occurs much less frequently. However, when I use the `large` model, this problem often occurs.）
This is how I am running Whisper in Colab:
**pip install -U openai-whisper
pip install setuptools-rust
!sudo apt update && sudo apt install ffmpeg
!whisper AAAAA.wav --model large --language zh --output_format srt**

**Comment by phineas-pta** [comment]
`large-v3` (default `large`) hallucinate a lot, u may want `large-v2` instead

**Reply by ioudove** [reply]
Yes, I don't hallucinate when I use medium
